releaser/make_release.py

```diff
@@ -289,6 +294,10 @@ def push(config):
         return
 
     chdir(config['repository'])
+    # rebase on upstream
+    doechocall('rebase on upstream before pushing main repository changes to GitHub',
```

Comment:
This probably does not hurt to do it here too, but this does NOT fix #15, as this rebases/push is for the "main" repository, not the feedstock repository, which is handled at: https://github.com/larray-project/releaser/blob/master/releaser/update_feedstock.py#L51